docs/demos/multi_page_basics/app.py

```diff
@@ -4,16 +4,14 @@
 
 app = Dash(__name__, plugins=[dl.plugins.pages])
 
-dash.register_page("home", layout="We're home!", path="/")
-dash.register_page("very_important", layout="Don't miss it!", path="/important", order=0)
-
+dash.register_page("another_home", layout="We're home!", path="/")
+dash.register_page(
+    "very_important", layout="Don't miss it!", path="/important", order=0
```

Comment:
maybe we wrap layout in html.Div or something so it's clear that this can be a dash set of components, not just a string